apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -348,51 +392,61 @@ export async function validateLti13CourseInstance(
 }
 
 export async function getAccessToken(lti13_instance_id: string) {
+  debug('getAccessToken');
   const lti13_instance = await selectLti13Instance(lti13_instance_id);
 
-  let tokenSet: TokenSet = lti13_instance.access_tokenset;
+  let tokenSet: client.TokenEndpointResponse | null = lti13_instance.access_tokenset;
```

Comment:
Nit, there's an opportunity for a Zod schema here to validate data as it comes out of the database.